src/leaflet.groupedlayercontrol.css

```diff
@@ -12,3 +12,23 @@
   overflow-y: scroll;
   padding-right: 10px;
 }
+
+.leaflet-control-layers-group.group-collapsable.collapsed .leaflet-control-layers-group-collapse,
+.leaflet-control-layers-group.group-collapsable:not(.collapsed) .leaflet-control-layers-group-expand,
+.leaflet-control-layers-group.group-collapsable.collapsed label:not(.leaflet-control-layers-group-label){
+    display: none;
```

Comment:
Can you switch to two-space tabs please? I know the repo doesn't have anything fancy to check for that (and might even have some inconsistencies due to other PRs 🙈 ), so try to match the existing indentation whenever possible.

Reply:
1) I designed the improvements to be disabled by default so that it doesn't cause problems with older configurations, so upgrading should have no problems.

2) I will try finding time in the next few days to revisit the code and make the requested changes.  I also want to try adding another space saving feature as well.